portfolio.css

```diff
@@ -0,0 +1,97 @@
+body {
+    /* uiGradient - Vanusa */
+    background: #da4453; /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #89216b, #da4453) fixed; /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #89216b,     #da4453) fixed; /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+.col-4 {
+    position: relative;
+    float: left;
+    width: 25%;
+    height: 300px;
+}
+
+.gallery {
+    overflow: auto;
+    width: 1200px;
+    margin: 0 auto;
+}
+
+.portfolio-image {
+    height: 200px;
+    width: 200px;
+    border-radius: 20%;
+    margin: 0 auto;
+    background-size: contain;
+
+    /* Vertically and horizontally centre div */
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.icons {
+    margin-top: 70px;
+    text-align: center;
+}
+
+.ie .gallery img {
+    top: 0;
+    left: 0;
+}
+
+/****************************
+Media Queries
+/****************************/
+
+@media screen and (max-width: 1340px) {
+
+    .col-4 {
```

Comment:
col-4 with a width of 33.3% is misleading
create new md and sm class to handle media queries
eg: col-md-3

Reply:
I've googled a lot about how to do this but can't figure it out, so as a workaround I renamed "col-4" to "grid-column" which is less misleading. Can you let me know how to replace one class with another in a media query? (e.g. replace col-lg-4 with col-md-3). I couldn't work out how to stop col-lg-4 applying and make col-md-3 apply with a media query. Thanks